src/KapetaAPI.ts

```diff
@@ -401,62 +400,48 @@ export class KapetaAPI {
             headers: {
                 'content-type': 'application/x-www-form-urlencoded',
                 accept: 'application/json',
-            },
+            } as any,
             method: 'POST',
             body: new URLSearchParams({
                 ...payload,
                 client_id: this.getClientId(),
-            }).toString(),
+            }) as any,
```

Comment:
Fetch types are funny, they only want `ReadableStream` inputs, even though the docs clearly say that a lot variety of types are handled.
> Any body that you want to add to your request: this can be a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob), an [ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer), a [TypedArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray), a [DataView](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView), a [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData), a [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams), string object or literal, or a [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream) object.

https://developer.mozilla.org/en-US/docs/Web/API/fetch#body